ansible/roles/outline/tasks/main.yml

```diff
@@ -76,11 +86,16 @@
       path: "{{ outline_config.project_source }}/.MIGRATION_PENDING"
       state: "absent"
 
+- name: "Install required packages"
+  ansible.builtin.apt:
+    name: "wget"
+    state: "present"
```

Comment:
Could you explain why this is necessary?

Reply:
Ah this was from the time I thought the health checks were performed externally, not within the container. I will remove this thanks!